codegateinc/react-smart-scroller
# tsdx setup

# Pull Request Template

## Description

It uses tsdx to provide esm,cjs and umd outputs.

Fixes https://github.com/codegateinc/react-smart-scroller/issues/61 (issue)

## Type of change

Please delete options that are not relevant.


## How Has This Been Tested?

1. Make sure new output is being generated.
![image](https://user-images.githubusercontent.com/9931428/79485425-8313c180-8015-11ea-83a3-3fbb177f794a.png)
2. Copy output to project which depends on this lib.
3. Build the app using `production` webpack config ex. via `react-scripts build` from CRA.
4. Check if bundle does't have any dead code in it.

## Checklist:
